app/ui/src/app/common/empty-state-card.component.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  Component
+} from '@angular/core';
+
```

Comment:
Why all these linebreaks here? We turn a simple import statement into 3 lines of code. This makes sense when the amount of imported tokens is so long than it does not fit in a single line without forcing the linter to complain.